saleor/payment/gateways/adyen/plugin.py

```diff
@@ -224,6 +226,17 @@ def __init__(self, *args, **kwargs):
         self.adyen = Adyen.Adyen(
             xapikey=api_key, live_endpoint_prefix=live_endpoint, platform=platform
         )
+        self._init_http_client()
+
+    def _init_http_client(self):
+        adyen_client = self.adyen.client
+        self.adyen.client.http_client = HTTPClient(
```

Comment:
since you have already defined `adyen_client` variable :
```suggestion
        adyen_client.http_client = HTTPClient(
```
in line 239 as well